packages/components/src/styles/container/container.stories.ts

```diff
@@ -133,3 +133,96 @@ export const CustomPadding = {
     });
   }
 };
+
+/**
+ * You can add a triangle indentation to the container using the `--triangle-` class appended with one of the following positions 'top', 'right', 'bottom', 'left' (e.g. `sd-container--triangle-top`).
+ */
+
+export const TrianglePosition = {
+  parameters: {
+    controls: {
+      exclude: ['sd-container--triangle-...', 'sd-container--variant-...']
+    }
+  },
+  render: (args: any) => {
+    return html` ${generateTemplate({
+      constants: [{ type: 'attribute', name: 'sd-container--variant-...', value: 'primary' }],
+      axis: {
+        y: [
+          {
+            type: 'attribute',
+            name: 'sd-container--triangle',
+            values: [
+              'sd-container--triangle-top',
+              'sd-container--triangle-right',
+              'sd-container--triangle-bottom',
+              'sd-container--triangle-left'
+            ]
+          }
+        ]
+      },
+      args
+    })}`;
+  }
+};
+
+/**
+ * For the `sd-container--variant-border-neutral-400`, an alternate set of classes must be used to create a triangle with a border.
+ */
+
+export const TriangleBorder = {
+  parameters: {
+    controls: {
+      exclude: ['sd-container--variant-...', 'sd-container--triangle-...']
+    }
+  },
+  render: (args: any) => {
+    return html` ${generateTemplate({
+      constants: [{ type: 'attribute', name: 'sd-container--variant-...', value: 'border-neutral-400' }],
+      axis: {
+        y: [
+          {
+            type: 'attribute',
+            name: 'sd-container--triangle',
+            values: [
+              'sd-container--triangle-top-border',
+              'sd-container--triangle-right-border',
+              'sd-container--triangle-bottom-border',
+              'sd-container--triangle-left-border'
+            ]
+          }
+        ]
+      },
+      args
+    })}`;
+  }
+};
+
+/**
+ * You can set the color of the triangle cut-out using the `--triangle-background` CSS property. CSS variables can be set either with an inline style: `style="--triangle-background: rgb(var(--sd-color-primary-600, 0 53 142) / 1);"` or a custom class:
+ * `.custom-sd-container {
+    --triangle-background: rgb(var(--sd-color-primary-600, 0 53 142) / 1);
+  }`
+ */
+
+export const TriangleColor = {
```

Comment:
Is there a specific reason (maybe related to the discussion with @coraliefeil about combining stories), why you only added one color? It looks like the container can only be used with primary blue and the triangle.

can we just add all colors here in terms of documentation?

Reply:
I asked about it above and received no feedback ^

"All colors" means the colors included as variants?

Reply:
new suggestion as discussed in the call.